skyportal/handlers/api/photometry.py

```diff
@@ -562,6 +562,10 @@ def insert_new_photometry_data(
                 .first()
             )
             group_ids = [public_group.id]
+
+        # always add the single user group of the uploading user
+        group_ids.append(self.associated_user_object.single_user_group.id)
```

Comment:
Maybe check to see if it wasn't explicitly specified, or just pass it to `set()` to take care of dups?

Reply:
fixed